src/components/DataTable/index.js

```diff
@@ -66,7 +69,7 @@ const DataTable = ({
   };
 
   return (
-    <Table className={style.formatTable}>
+    <Table className={headerStyle ? style.formatTable : style.anotherFormatTable}>
```

Comment:
please change the classname "anotherFormatTable" into something more descriptive